governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+>
+> **Trustee has no discretion for executions outside of Governance Poll approved instruction sets.** [emphasis added]
+
+There are three problems related to the Standing Orders in the Exercise Prompt, which run counter to both aspects of my mandate as a Governance Facilitator.
+
+First, the proposed Standing Order does not offer voters enough information to make an informed decision; this is due to the ambiguities in the proposed Standing Order, which I assume are non-trivial. If the Standing Order/Instruction Set is to be akin to safely executable code, it cannot be ambiguous without presenting significant risk to Maker. 
+
+For the Clydesdale RWA structure, a library of standing orders/instruction sets was proactively developed and validated by the pertinent Maker ecosystem actors prior to any immediate need. In the immediate situation, we are apparently missing the expert input/assessment of such ecosystem actors, including the Strategic Finance Core Unit. (SFCU's [mandate](https://mips.makerdao.com/mips/details/MIP55c3SP12#sentence-summary) is the monitoring of RWA vaults and includes the review of legal structures, resolutions, wordings and agreements.) From the Exercise prompt, I assume this oversight may be due to Emilia's mistaken belief that her conversations with "large MKR governance holders" sufficed, without further ado, to bring the proposal forward. (See [here](https://forum.makerdao.com/t/proposal-to-increase-debt-ceiling-for-mip65-by-750m-to-1250m/20060#makerdao-resolution-instruction-sets-12) for another pertinent Clydesdale precedent, announcing that a proposed instruction set will be shared along *with* an "assessment by a relevant member with legal credentials in the Maker Community.")
+
+Second, the proposal purports to unilaterally invalidate the current Standing Order concerning ALM transfers. Curiously, the proposed Standing Order concerns a different topic entirely (revenue return) than the one Emilia purports to invalidate. Emilia's attempted unilateral invalidation of a current Standing Order (and one involving Maker's first line of defense of the peg, no less) crosses the line that is forbidden to the Trustee - an "execution outside of a Governance approved instruction set." It is also arguably a violation of the [Stability Scope provision](https://mips.makerdao.com/mips/details/MIP104#5-1-arrangers) prohibiting Arrangers from "operat[ing] or influenc[ing] the arranged legal and operational structure’s asset operations in a way that can cause any significant harm or losses to Dai’s stability." Even *if* the replacement ALM order were already drafted and ready to be presented, nevertheless, the current Standing Order cannot be unilaterally done away with. Only an official act of Maker Governance can alter or replace Standing Orders. 
```

Comment:
"Second, the proposal purports to unilaterally invalidate the current Standing Order concerning ALM transfers. Curiously, the proposed Standing Order concerns a different topic entirely (revenue return) than the one Emilia purports to invalidate." - Yes, I know they don't actually work that way. Making good exercises is hard.

Reply:
"Even *if* the replacement ALM order were already drafted and ready to be presented, nevertheless, the current Standing Order cannot be unilaterally done away with. Only an official act of Maker Governance can alter or replace Standing Orders."

Worth noting that voting in the new standing order (which could include text invalidating the previous ones (I think)) would serve as an official act of Maker governance. Alternatively, it could be explicitly polled to revoke the older one.

Reply:
"Emilia's attempted unilateral invalidation of a current Standing Order (and one involving Maker's first line of defense of the peg, no less) crosses the line that is forbidden to the Trustee - an "execution outside of a Governance approved instruction set."

You're going down a route here I didn't really intend. But the important part is that you're actually seriously thinking about it.